### ISSUE TITLE
Support `stopEarly`

### ISSUE THREAD

**Issue by julien-f** [jorgebucaran/getopts#40]
Hello,

It would be nice to support a `stopEarly` option, similar to [minimist](https://github.com/substack/minimist) which would stop parsing at the first non-option argument.

This is especially useful when defining subcommands in a CLI:

```js
function foo (args) {
  // handle foo options
  const options = getopts(args)
}

function main () {
  const options = getopts(process.argv.slice(2), {
    stopEarly: true,
  })

  const [subcommand, subargs] = options._
  if (subcommand === 'foo') {
    return foo(subargs)
  }
}
```

**Comment by jorgebucaran** [comment]
@julien-f Sounds good. I'll start working on it soon. 💯

**Comment by julien-f** [comment]
Thank you :slightly_smiling_face: 

I'm not in any hurry though :wink:

**Comment by jorgebucaran** [comment]
Available in 2.2.3 ae5900d ([Notes](https://github.com/jorgebucaran/getopts/releases/tag/2.2.3)). 🎉
